app/views/users/new.html.erb

```diff
@@ -0,0 +1,33 @@
+<h1>User#new Form</h1>
+<%# <form method='POST' action='/users' accept-charset="UTF-8"> %>
+<%# <input type="hidden" name="authenticity_token" value="<%= form_authenticity_token %>
+<%# <label for='usename'>Username: </label> %>
+<%# <input type='text' name='user[username]' placeholder='Enter your username'><br> %>
+<%# <label for='email'>Email: </label> %>
+<%# <input type='email' name='user[email]' placeholder='Enter your email'><br> %>
+<%# <label for='password'>Password: </label> %>
+<%# <input type='password' name='user[password]' placeholder='Enter password at least 8 alphanum'><br> %>
+<%# <input value='submit' type='submit'> %>
+<%# </form> %>
```

Comment:
- This form is not properly commented out. A simple trick is to highlight the code then press and hold `CTRL+/`
Below is a properly commented out code:
```ruby
# <%= form_with model: @user do |form| %>
#     <%= label_tag(:username, "Username:") %><br>
#     <%= form.text_field :username %><br><br>
#     <%= label_tag(:email, "email:") %><br>
#     <%= form.email_field :email %><br><br>
#     <%= label_tag(:password, "Password:") %><br>
#     <%= form.password_field :password %><br><br>
#     <%= form.submit :Submit %>
# <% end %>
```